Build/package/cpptools_extension_pack.yml

```diff
@@ -0,0 +1,48 @@
+name: $(date:yyyyMMdd)$(rev:.r)
+trigger: none
+pr: none
+
+parameters:
+- name: verifyVersion
+  displayName: Attest version in package.json is correct
+  type: boolean
+  default: false
+- name: verifyChangelog
+  displayName: Attest CHANGELOG.md is updated
+  type: boolean
+  default: false
+
+resources:
+  repositories:
+  - repository: MicroBuildTemplate
+    type: git
+    name: 1ESPipelineTemplates/MicroBuildTemplate
+    ref: refs/tags/release
+
+extends:
+  template: azure-pipelines/MicroBuild.1ES.Official.yml@MicroBuildTemplate
+  parameters:
+    pool:
+      name: AzurePipelines-EO
+      image: AzurePipelinesWindows2022compliantGPT
+      os: windows
+    sdl:
+      sourceAnalysisPool:
+        name: AzurePipelines-EO
+        image: AzurePipelinesWindows2022compliantGPT
+        os: windows
+
+    stages:
+    - stage: package
+      jobs:
+      # Introduce pipeline compilation errors to block scheduling if the requisite parameters are not set
+      - ${{ if not(eq(parameters.verifyVersion, true)) }}:
+        - 'Binary version in CMakeLists.txt should be updated before scheduling the pipeline.'
```

Comment:
Is "binary version" correct? Or should that be the package.json version?

Reply:
Thanks. I copied from the cpptools pipeline, but missed updating these parts.